src/index.scss

```diff
@@ -232,3 +232,7 @@ header {
   position: sticky;
   top: 0;
 }
+
+.color-00688d {
+  color: #00688D !important;
+}
```

Comment:
This change is unnecessory because the link will have the same color code after applying the theme.